src/components/common/course-enrollments/CourseEnrollments.jsx

```diff
@@ -19,21 +19,20 @@ import * as selectors from './data/selectors';
 import * as actions from './data/actions';
 
 export class CourseEnrollments extends Component {
-  static contextType = LayoutContext;
+  static contextType = AppContext;
 
   componentDidMount() {
     const {
       pageContext: {
-        pageType,
         programUUID, // for Masters, empty for Enterprise
         enterpriseUUID, // for Enterprise, empty for Masters
       },
     } = this.context;
     const { fetchCourseEnrollments } = this.props;
-    const options = { pageType };
-    if (pageType === 'pages.ProgramPage') {
+    const options = {};
+    if (programUUID) {
```

Comment:
It's not really necessary to check based on `pageType` here. Instead, we can rely on the existence of `programUUID` and `enterpriseUUID` variables.